erpnext/accounts/doctype/journal_entry/journal_entry.py

```diff
@@ -569,14 +569,18 @@ def get_default_bank_cash_account(company, account_type=None, mode_of_payment=No
 		if account_type=="Bank":
 			account = frappe.db.get_value("Company", company, "default_bank_account")
 			if not account:
-				account = frappe.db.get_value("Account",
-					{"company": company, "account_type": "Bank", "is_group": 0})
+				account_list = frappe.get_all("Account", filters = {"company": company,
+					"account_type": "Bank", "is_group": 0})
+				if len(account_list) == 1:
```

Comment:
Should be

```
if len(account_list) > 0:
```

Also please show proof

Reply:
Actually the issue is that if there are multiple bank type account, firstly created account shouldn't be chooosed (in Journal Entry). If there is only one bank type account we can set it.